src/app/services/baw-api/account/accounts.service.ts

```diff
@@ -40,7 +42,21 @@ export class AccountsService extends StandardApi<User> {
     return this.apiFilter(endpoint(Empty, Filter), filters);
   }
   public show(model: IdOr<User>): Observable<User> {
-    return this.apiShow(endpoint(model, Empty));
+    return this.apiShow(endpoint(model, Empty)).pipe(
+      // Return unknown or deleted user depending on error code
+      catchError((err: ApiErrorDetails) => {
+        switch (err.status) {
+          case apiReturnCodes.unauthorized:
+            console.warn("Returning Unknown User");
+            return of(User.unknownUser);
+          case apiReturnCodes.notFound:
+            console.warn("Returning Not Found User");
```

Comment:
these probably don't need to be logged - maybe better to add comments and explain the situations for which these occur a little more.

- not found: only occurs when a user if soft-deleted
- unknown: only occurs when user is anonymous to server